### PR TITLE
Add rail Delay-repay usage example

### PR DESCRIPTION
**Pull request by lddelchev**
Ticket: https://kanbanzone.io/b/YA2t2Uox/c/1075-Rail-Performance-Delay-repay-Usage-Example

Usage example to show basic usage of the `train/station_actual_journeys/{station}.json` endpoint in a delay-repay app.

See on staging: https://developer.transportapi.com/examples-staging?example=train-delay-repay&branch=rail-performance

### CODE REVIEW

File: src/examples/bus-journey-patterns/main.js [hunk 1]
```diff
@@ -37,5 +37,6 @@ function tableRowHtml (atcocodeToStopName, journeyPatterns, rowIndex) {
 }
 
 function tableCellHtml (atcocodeToStopName, journeyPattern, rowIndex) {
-  return `<td><b>${atcocodeToStopName[journeyPattern.stops[rowIndex]?.atcocode] || ''}</b> ${journeyPattern.stops[rowIndex]?.atcocode || ''}</td>`
```
Comment (vdimchev) [inline src/examples/bus-journey-patterns/main.js:40]: What's the purpose of this split? Readibility, linter god?

File: src/examples/train-delay-repay/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "places-list",
```
Comment (vdimchev) [inline src/examples/train-delay-repay/package.json]: Probably not `places` ;)

**Review by vdimchev — CHANGES_REQUESTED** [review]
One small thing.

File: src/examples/bus-journey-patterns/main.js (hunk 1 above)
Comment (lddelchev) [reply src/examples/bus-journey-patterns/main.js:40]: We were using safety navigation operator (`?.`) which is not supported by the version of node that Codesandbox use and the rendering of the example on their page was broken.

File: src/examples/train-delay-repay/package.json (hunk 2 above)
Comment (lddelchev) [reply src/examples/train-delay-repay/package.json]: Fixed along with other copy-pasted `package.json` files :)

File: src/examples/bus-journey-patterns/main.js (hunk 1 above)
Comment (vdimchev) [reply src/examples/bus-journey-patterns/main.js:40]: Yes, I was asking exactly because I saw it's still used even after the split.

**Review by vdimchev — APPROVED** [review]

**Review by katya-spasova — APPROVED** [review]

**Pull request merged**